gno.land/pkg/sdk/vm/gas_test.go

```diff
@@ -67,7 +67,7 @@ func TestAddPkgDeliverTx(t *testing.T) {
 	gasDeliver := gctx.GasMeter().GasConsumed()
 
 	assert.True(t, res.IsOK())
-	assert.Equal(t, int64(87809), gasDeliver)
+	assert.Equal(t, int64(88809), gasDeliver)
```

Comment:
interesting; 200 is the price for the extra check? pretty cheap :)

Reply:
The GnoSDK's performance is potentially very good at the moment. This is a partial proof, missing the cost of creating a Gno.machine, that suggests its potential. cc @zivkovicmilos